chapter1_welcome/chapter1_welcome.md

```diff
@@ -0,0 +1,153 @@
+# Chapter 1 Welcome Your Journey Starts Here
+
+Welcome to the Topcon Sitelink3D v2 API example repository documentation. The chapters contained in this repository make reference to the Python scripts found in our Sitelink3D v2 Examples [here](https://github.com/Sitelink3D-v2-Developer/sitelink3dv2-examples). 
+
+If you’re feeling daunted with where to start with our API and are looking for a guided walk through of the concepts, then you’re in the right place. We know what it’s like starting to learn a new system and its associated API. The swagger can be too detailed to start with and some of the high-level material sometimes seems more like marketing than a coding tool. That’s where this documentation fits in.
+
+In writing this walkthrough, we are assuming a technical audience with construction domain knowledge but minimal experience with Sitelink3D v2. Concepts will be covered sequentially with sufficient detail to get you using the API.
+So make a cup of coffee and let’s get started!
+## Documentation Structure
```

Comment:
Documentation structure should be "Documentation Chapters". We could call this .md file a preface.

Reply:
Going with this idea, we could add numbering points to each chapter subheading.